experimental/packages/opentelemetry-sdk-metrics-base/src/view/Aggregation.ts

```diff
@@ -24,19 +28,91 @@ import { InstrumentDescriptor } from '../InstrumentDescriptor';
  * Aggregation provides a set of built-in aggregations via static methods.
  */
 export abstract class Aggregation {
```

Comment:
A name like `AggregationSelector` may make it more obvious what this is doing

Reply:
Yes, it does work like `AggregationSelector` in the meaning.